_posts/en/newsletters/2021-10-13-newsletter.md

```diff
@@ -0,0 +1,249 @@
+ancestor feerate, so if we allowed new inputs to be added, a replacement
+transaction could be less attractive to mine than the transaction it is
+replacing."
+  a2link="https://bitcoincore.reviews/22675#l-52"
+
+  q3="In BIP125 Rule #4, what does it mean for a transaction to \"pay for its
+own bandwidth?\" Why don’t we just allow any replacement as long as it has a
+higher feerate?"
+  a3="\"Paying for its own bandwidth\" means paying a fee which includes an
+additional amount that would cover the minimum relay fee for the replacement
+transactions. Without this rule, malicious actors could repeatedly bump their
+transaction fee by 1 satoshi, and use a disproportionate amount of mempool
+compute resources and network bandwidth."
+  a3link="https://bitcoincore.reviews/22675#l-117"
+
+  q4="Replace-by-fee logic is concerned with mempool policy. Why does [this
+logic][transaction spends its outputs] return that the replacement transaction
+failed due to consensus rules, not policy rules?"
+  a4="This logic catches the case where the replacement transaction is spending
+a transaction that it is replacing. Since it's impossible for both the original
+transaction and the replacement transaction to be confirmed, this replacement
+transaction can never appear in the block chain, and so is consensus invalid."
+  a4link="https://bitcoincore.reviews/22675#l-40"
+
+%}
+
+## Preparing for taproot #17: is cooperation always an option?
+
+*A weekly [series][series preparing for taproot] about how developers
+and service providers can prepare for the upcoming activation of taproot
+at block height {{site.trb}}.*
+
+{% include specials/taproot/en/17-keypath-universality.md %}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Eclair 0.6.2][] in a new release that includes a fix for
+  the vulnerability described in the *news* section above as well new
+  features and other bug fixes described in its [release notes][eclair
+  rn].
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[BDK][bdk repo], [Bitcoin Improvement Proposals (BIPs)][bips repo], and
+[Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #20487][] adds a `-sandbox` configuration option that
+  can be used to enable an experimental system call (syscall) sandbox.
+  When the sandbox is active, the kernel will terminate Bitcoin Core if
+  it makes any syscalls other those on a per-process whitelist.  The
+  mode is currently only available on x86_64 and is mainly meant for
+  testing.
```

Comment:
I think this helps provide context/motivation for this PR (paraphrased from the commit), "as this feature allows easier to reasoning about the capabilities required of each type of thread in Bitcoin Core."